candi/cloud-providers/yandex/terraform-modules/master-node/variables.tf

```diff
@@ -49,7 +49,7 @@ locals {
   platform              = lookup(local.master_instance_class, "platform", "standard-v2")
   cores                 = local.master_instance_class.cores
   memory                = local.master_instance_class.memory / 1024
-  disk_size_gb          = lookup(local.master_instance_class, "diskSizeGB", 20)
+  disk_size_gb          = lookup(local.master_instance_class, "diskSizeGB", 50)
```

Comment:
Should we change this?

Reply:
I cannot understand, how we should converge state of existing clusters ? Manually ? Maybe we need a migration to set diskSizeGB to 20 in existing installations ?

Reply:
Indeed. Let it be a separate decision